src/classes/AFFL_MultiRecordType_TDTM.cls

```diff
@@ -74,8 +74,18 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
 				for (SObject so : newlist) {
 					Affiliation__c affl = (Affiliation__c)so;
 
-					String lookupFieldLabel = afflMapper.mapAccRecTypeToContactLabel.get(affl.Affiliation_Type__c);
-					String lookupFieldName = afflMapper.contactLabelNames.get(lookupFieldLabel);
+					String lookupLabelOrAPINameFromMapping = afflMapper.mapAccRecTypeToContactLabel.get(affl.Affiliation_Type__c);
+					String lookupFieldName = afflMapper.contactLabelNames.get(lookupLabelOrAPINameFromMapping);
+          
+					// If lookupFieldName of the Contact is not found, check if lookupLabelOrAPINameFromMapping is an API, if yes assign it to lookupFieldName
+					// This code block below is to make sure that Primay Affiliation lookup is populated on contact even when language is not English
+						
+          if ((lookupLabelOrAPINameFromMapping != NULL && lookupLabelOrAPINameFromMapping != '') && (lookupFieldName == '' || lookupFieldName == NULL)) {
```

Comment:
There seems to be a spacing issue here:

![Screen Shot 2019-10-28 at 1 34 17 PM](https://user-images.githubusercontent.com/1894238/67707069-d28f0400-f987-11e9-83c9-3542027aa29a.png)